app/controllers/posts_controller.rb

```diff
@@ -0,0 +1,45 @@
+class PostsController < ApplicationController
+
+  def index
+    @posts = Post.all
+  end
+
+  def show
+    @post = Post.find(params[:id])
+  end
+
+  def new
+    puts "This is a new post"
+    @post = Post.new
+  end
+
+  def create
+    puts params
+    puts "New Post Created"
```

Comment:
Totally fine to do "puts" or use `pry` when your debugging, just remember to take it out after!